.github/workflows/set-version.yaml

```diff
@@ -26,15 +26,15 @@ jobs:
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/typescript.d.ts
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/tsserverlibrary.d.ts
         sed -i -e 's/const version\(: string\)\{0,1\} = .*;/const version = "${{ github.event.client_payload.package_version }}" as string;/g' src/compiler/corePublic.ts
-        npm ci
+        npm install
         npx hereby LKG
         npm test
         git diff
-        git add package.json
+        git add package.json package-lock.json
         git add src/compiler/corePublic.ts
         git add tests/baselines/reference/api/typescript.d.ts
         git add tests/baselines/reference/api/tsserverlibrary.d.ts
-        git add ./lib
+        git add --force ./lib

```

Comment:
Actually, do we run this on main? If we run this on main to bump the version, then this isn't a good idea. @DanielRosenwasser what do you do when you bump main's version?

Reply:
I do not run `set-version` on `main`, I set it manually.